manifests/deis-builder-service.yaml

```diff
@@ -12,6 +12,10 @@ spec:
       targetPort: 2223
       name: ssh
       protocol: TCP
+    - port: 3000
+      targetPort: 3000
```

Comment:
this is redundant, so let's remove this: https://github.com/deis/charts/issues/3